Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -161,6 +162,44 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcFails()
         }
 
         [TestMethod]
+        public void Exe_PreProc_InstallTargets()
```

Comment:
The name isn't very informative. It looks like this test is checking that the targets are not installed if the switch is set.